### PR TITLE
clarify requirements for length in future extentions

### LINKED ISSUE #46: Specify EKTMsgLength as a requirement on the sender of EKT message
EKTMsgLength  All EKT message other that ShortEKTField must have a
      length as second from the last element.  This is the length in
      octets of either the FullEKTField/ExtensionEKTField including this
      length field and the following message type.

>>> This "must" statement is really a requirement on the person that
>>> will specify other EKT messages.  Is there a better place to state
>>> This requirement?

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #46

### CODE REVIEW

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 1]
```diff
@@ -875,6 +875,12 @@ SRTP packets                 <------->      SRTP packets
         allocation of even values over odd ones until the even code
         points are consumed to avoid conflicts with pre standard
         versions of EKT that have been deployed.</t>
+
+        <t> All new EKT message MUST be defined to have a length as
```
Comment (paulej) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: "message" should be plural as phrased

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 2]
```diff
@@ -875,6 +875,12 @@ SRTP packets                 <------->      SRTP packets
...
+        second from the last element so that implementations that do
+        not understand the new extension can still skip over it when
```
Comment (paulej) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: I'd remove the word "still"

File: srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml [hunk 3]
```diff
@@ -875,6 +875,12 @@ SRTP packets                 <------->      SRTP packets
...
+        parsing the message. <t>
```
Comment (paulej) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: Actually, having the length there for extensions is only one reason. Another is that this obviates the need for successful processing to be dependent on having knowledge of the SRTP master key length.  While this is usually known by receiving endpoint, some scenarios we discussed in PERC, like having a null HBH cipher presented problems for media distributors.

Comment (fluffy) [inline srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml]: following your advice of "it is what is" and less explanation of the design, I will drop the 2nd half of the sentence

**Pull request merged**